===================================================================
RCS file: /usr/local/src/cvsroot/nefu/README,v
retrieving revision 1.11
retrieving revision 1.12
diff -u -r1.11 -r1.12
--- README	2002/07/03 17:40:06	1.11
+++ README	2003/11/19 16:01:13	1.12
@@ -9,5 +9,9 @@
 
 The file INSTALL contains instructions on compilation and installation.
 
+The directory TDK contains nefu's Test Development Kit.  The TDK
+is an aid for adding new shell and binary tests to nefu.  The file
+TDK/README contains instructions for its use.
+
 nefu is maintained at the University of Michigan.  Please mail nefu@umich.edu
 with all suggestions or bugs.
